nautobot/ipam/models.py

```diff
@@ -971,23 +972,6 @@ def _deconstruct_address(self, address):
             self.mask_length = address.prefixlen
             self.ip_version = address.version
 
-    def get_duplicates(self):
-        return IPAddress.objects.filter(vrf=self.vrf, host=self.host).exclude(pk=self.pk)
-
-    # TODO: The current IPAddress model has no appropriate natural key available yet.
-    natural_key_field_names = ["id"]
```

Comment:
What is the natural/composite key for IPAddress now?

Reply:
```python
In [1]: IPAddress.natural_key_field_lookups
Out[1]: ['parent__namespace__name', 'parent__network', 'parent__prefix_length', 'host']
```

Reply:
Interesting, I'd almost expect that the natural key for an IPAddress should be simply (`parent__namespace__name`, `host`)

Reply:
IPAddress.natural_key_field_lookups was overloaded to `['parent__namespace__name', 'host']`.